docs/IE_PLUGIN_DG/NewTransformation.md

```diff
@@ -1,15 +1,386 @@
 # Writing ngraph transformations {#new_ngraph_transformation}
 
-1. Code of such transformation MUST be directly in template plugin soruces.
-2. It must be mark with doxygen markeds like
+This guide contains all necessary information that could help you to start writing nGraph transformations.
 
+First of all before writing transformation make sure that there is no transformation with the same functionality in [transformation library].
```

Comment:
can we fix the link `[transformation library]`? you have described some transformations already in this PR.

Reply:
Done.